pyccel/ast/operators.py

```diff
@@ -125,13 +142,15 @@ def broadcast(shape_1, shape_2):
 class PyccelOperator(TypedAstNode):
     """
     Abstract superclass for all builtin operators.
-    The __init__ function is common
-    but the functions called by __init__ are specialised
+
+    Abstract superclass for all builtin operators.
+    The __init__ function is common but the functions
+    called by __init__ are specialised.
 
     Parameters
     ----------
-    args: tuple
-        The arguments passed to the operator
+    *args : tuple
+        The arguments passed to the operator.
```

Comment:
Do we have a common type for all the elements of `args`?

Reply:
Same here:
```suggestion
    *args : tuple of TypedAstNode
        The arguments passed to the operator.
```